### PR TITLE
Added Spring to project

### PR DESCRIPTION
**Pull request by Andrewmazyar**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -42,6 +44,71 @@
...
+
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-core</artifactId>
+            <version>${spring.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-profile</artifactId>
```
Comment (boroda4436) [inline pom.xml:56]: do you need it?

File: pom.xml [hunk 2]
```diff
@@ -42,6 +44,71 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-webmvc</artifactId>
+            <version>${spring.version}</version>
+
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-dbcp2</artifactId>
```
Comment (boroda4436) [inline pom.xml:82]: why do we need this dependency?

File: pom.xml [hunk 3]
```diff
@@ -89,4 +164,5 @@
             </plugins>
         </pluginManagement>
     </build>
-</project>
+
+</project>
```
Comment (boroda4436) [inline pom.xml:168]: empty line

File: src/main/java/web/cinema/controllers/HelloController.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/user")
+public class HelloController {
+    @GetMapping("/hello")
+    public String sayHello() {
+        return "Hello_Bohdan_How_Zhittya";
```
Comment (boroda4436) [inline src/main/java/web/cinema/controllers/HelloController.java:12]: ахахах) Непогано) Дякую!

File: pom.xml (hunk 2 above)
Comment (Andrewmazyar) [reply pom.xml:82]: that connect file db.properties

**Pull request merged**